### DISCUSSION TITLE
support SPDK NVMe

### DISCUSSION THREAD

**Discussion by sqiu110** [alibaba/PhotonLibOS#152] [General]
will it consider supporting SPDK NVMe driver? right now, only io_uring is supported.

**Comment by lihuiba** [comment]
Yes.

Given that we already have support for DPDK (and user-space TCP, via f-stack), I believe SPDK will not be difficult. But we are short of hands for it at present, so there is not a schedule for it yet.

**Reply by lihuiba** [reply]
There's already a related issue: https://github.com/alibaba/PhotonLibOS/issues/119
